setup_aws_profiles.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
I guess this is OK... How were we writing these configs before? It would be better to write the secrets separately for each branch. The way it works in this PR, anyone who can push to the `develop` branch would get secrets for the prod accounts... which is fine today.

Reply:
Khai can just check the value of `$TRAVIS_BRANCH` and do different things for each. But is this overkill? These files are ephemeral.

Reply:
done, changed to configure secrets per job.